ftdetect/set_repl_cmd.vim

```diff
@@ -92,8 +92,15 @@ if has('nvim') || has('terminal')
           \ end
     " Clojure
     au FileType clojure
-          \ if executable('lein') |
+          \ if executable('lein') && filereadable('project.clj') |
```

Comment:
Since lein project is depends on 'project.clj' file, check if that file exists. same on clj-tool project.